api/clients/cert.go

```diff
@@ -0,0 +1,13 @@
+package clients
+
+import "github.com/Layr-Labs/eigenda/api/grpc/common"
+
+type Cert struct {
```

Comment:
Shouldn't this struct contain all fields of the blob status reply? It looks like the inclusion proof has been dropped (not sure if there's anything else.)

Reply:
The immediate need was to generalize across stage 0 integrations, and then add support for a stage 1 cert next. But will generalize across both stage 0, 1 of integrations. Stage 2 integrations require an aggregated signature in addition to the existing fields, so I've been operating under the assumption that the different stages will require different sets of fields.